.docker/release/files/etc/nginx/nginx.conf

```diff
@@ -52,7 +52,6 @@ http {
 
   server {
     listen 8888;
-    listen [::]:8888;
```

Comment:
Yep, did so here

Reply:
If the tests pass it should be good.

Reply:
OK, got it. Everything in `http.d` is not included.

Reply:
yes 😄